### PR TITLE
HPCC-15846 Prerm script change for dafilesrv stop

### PR DESCRIPTION
**Pull request by Michael-Gardner**
Signed-off-by: Michael Gardner michael.gardner@lexisnexis.com

### CODE REVIEW

**Comment by Michael-Gardner** [comment]
@richardkchapman re-target.
@xwang2713 please review.

File: initfiles/sbin/prerm.in [hunk 1]
```diff
@@ -28,7 +28,7 @@ fi
 # Stop all services before cleaning up
 ######################################
 /etc/init.d/hpcc-init stop
-if [ -f "${PID_DIR}/mydafilesrv.pid" ]; then
+if /etc/init.d/dafilesrv status >/dev/null 2>&1; then
```
Comment (richardkchapman) [inline initfiles/sbin/prerm.in]: Not sure why it needs to be conditional? Why not just unconditionally stop it as we do for hpcc-init ?

**Comment by Michael-Gardner** [comment]
@richardkchapman removed the conditional

**Comment by xwang2713** [comment]
Looks good

**Pull request merged**